tasks/main.yml

```diff
@@ -1,24 +1,32 @@
 ---
-- name: Install monasca-thresh
-  apt: name=monasca-thresh state=latest
-  notify:
-    - restart monasca-thresh
-
 - name: setup monasca_group
   group: name={{monasca_group}} system=yes
 
+- name: Setup user
+  user: name={{thresh_user}} system=yes group={{monasca_group}}
+
 - name: create conf_dir
   file: path={{monasca_conf_dir}} state=directory owner=root group={{monasca_group}} mode=775
 
-- name: Add thresh user to the monasca_group
-  user: name={{thresh_user}} append=yes groups={{monasca_group}}
+- name: create jar_dir
+  file: path={{monasca_jar_dir}} state=directory owner=root group=root mode=755
+
+- name: Fetch thresh jar
+  get_url: dest={{monasca_jar_dir}}/monasca-thresh.jar url="{{tarball_server}}/ci/monasca-thresh/monasca-thresh-{{thresh_version}}-shaded.jar"
```

Comment:
It would be good to change tarball_server to tarball_base_url or something similar and then put the directory path in the variable also. This is just because some of the mirrors I have used don't necessarily replicate the entire path.